src/plugins/telemetryTable/components/table-configuration.vue

```diff
@@ -22,22 +22,22 @@
                 </div>
             </li>
         </ul>
-        <div class="c-properties__header">
+        <div class="c-inspect-properties__header">
             Table Column Visibility
         </div>
-        <ul class="c-properties__section">
+        <ul class="c-inspect-properties__section">
             <li
                 v-for="(title, key) in headers"
                 :key="key"
-                class="c-properties__row"
+                class="c-inspect-properties__row"
             >
                 <div
-                    class="c-properties__label"
+                    class="c-inspect-properties__label"
                     title="Show or hide column"
                 >
                     <label :for="key + 'ColumnControl'">{{ title }}</label>
                 </div>
-                <div class="c-properties__value">
+                <div class="c-inspect-properties__value">
                     <input
                         :id="key + 'ColumnControl'"
```

Comment:
looks like this was already here but i prefer using reactive properties (method in this case) and in this case you could reuse for both bindings

Reply:
I won't change this since it was already here.